cfgov/unprocessed/js/routes/common.js

```diff
@@ -7,8 +7,6 @@
 
 // Global modules.
 require( '../modules/focus-target' ).init();
-require( '../modules/UStreamPlayer' ).init( '.video-player__ustream' );
```

Comment:
Is this not needed anymore? It's removed here but not copied anywhere else.

Reply:
yeah, I'll remove it in another PR because it's going to require removing some python code and generating a migration.

Reply:
Got it!